packages/parse5-htmlparser2-tree-adapter/package.json

```diff
@@ -26,7 +26,7 @@
         "parse5": "^7.0.0"
     },
     "scripts": {
-        "build:cjs": "tsc --moduleResolution node10 --module CommonJS --target ES6 --outDir dist/cjs && echo '{\"type\":\"commonjs\"}' > dist/cjs/package.json"
+        "build:cjs": "tsc --moduleResolution node16 --module node16 --target ES6 --outDir dist/cjs && echo '{\"type\":\"commonjs\"}' > dist/cjs/package.json"
```

Comment:
im not sure you can change `module` here since `node16` will detect that it is `type: "module"` and output ESM

`--module CommonJS` was forcing it to output commonjs regardless of the `type`

Reply:
Thanks for the review 🙏

Node10 module resolution wasn't working after the change, and module CommonJS can't be paired with Node16 resolution.

The other PR ran into the same thing.

I'll find a proper solution tomorrow.

Reply:
I looked into migrating to tshy to resolve, which seemed to do the trick.